build/repo.targets

```diff
@@ -1,5 +1,24 @@
 <Project>
+  <PropertyGroup>
+    <PrepareDependsOn>$(PrepareDependsOn);Internalize</PrepareDependsOn>
```

Comment:
This should be prepended to CompileDependsOn. Because Internalize depends on Restore, adding it here puts restore before Prepare, which may break the [default korebuild lifecycle](https://github.com/aspnet/KoreBuild/blob/dev/build/targets/KoreBuild.Common.targets).

```xml
<CompileDependsOn>Internalize;$(CompileDependsOn)</CompileDependsOn>
```